src/main/java/lotto/domain/Winning.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Winning {
+    private Map<Rank, Integer> winnerCountMap;
+
+    public Winning() {
+        this.winnerCountMap = new HashMap<>();
+    }
+    public void addWinning(int rightNumber, boolean bonusMatch) {
+        if (Rank.isWinning(rightNumber)) {
+            final Rank rank = Rank.ofRightNumber(rightNumber, bonusMatch);
+            winnerCountMap.put(rank, winnerCountMap.getOrDefault(rank, 0) + 1);
+        }
+    }
+
+    public int getWinnerCount(Rank rank) {
+        return winnerCountMap.getOrDefault(rank, 0);
+    }
+
+    public double getReturnRate(final Amount purchaseAmount) {
+        Amount sum = Amount.ZERO;
+        for (Rank rank : winnerCountMap.keySet()) {
+            final Amount rankPrice = rank.getAmount();
+
+            final int count = winnerCountMap.get(rank);
+            sum = rankPrice.multiply(count);
```

Comment:
```suggestion
sum = rank.multiply(winnerCountMap.get(rank));
```

상태를 가지고있는 객체에게 곱하라고시켜도 좋을것 같네요~

Reply:
Amount객체에 plus를 추가하고, Rank내부에서 본인의 상금을 곱하는 로직을 추가하였습니다. 
상태를 가지고 있는 객체라는게 혹시 Rank라는 상수가 상태를 가지는걸 뜻하는걸까요???